### ISSUE TITLE
Foundation 4 from 'create project' where are the Foundation 4 SCSS partial scss??

### ISSUE THREAD

**Issue by JTBonGITHUB** [KKBOX/CompassApp#187]
I've searched/googled everywhere for this, and cannot find an answer.
SYSTEM: Win 7 64-bit
COMPASS.APP version: 1.26
I have not installed Compass via the command line or via Ruby, so there should be no conflict from that perspective.
IDE: Dreamweaver CS5.5 and/or others if needed.

ISSUE: I basically understand how SCSS works within the Compass.app. I created a ZURB FOUNDATION 4.1.5 project.
However, there is not a FOLDER called FOUNDATION (or something else) within the PROJECT FOLDER that contains the Foundation 4 partial (component) SCSS files (those that begin with _ ).   Examples of the files I'm referencing are:
_alert-boxes
_block-grid

These files DO exist within the Compass.app lib/ruby/compass_extensions/zurb-foundation-4.1.5/scss/foundation/components folder.

But these are the ORIGINAL ones included w/ the Compass.app, so I don't want to alter these, correct?

Why were copies of these not made and placed in the SASS folder, or a new FOUNDATION folder??

In short, the INDEX.HTML file is controlled by the app.css file, which is 'updated/compiled/ via the app.scss file (which exists in my PROJECT website I created w/ the Compass.app). The app.scss file has @import for the 'settings' file (which I'm assuming is the _settings.scss file that is also located in my PROJECT website) and it has @import for the 'foundation' file - which is located......where????  There are of course commented-out @imports for the other Foundation components files - which are located....where??

I don't want to alter the ORIGINAL files that came with Compass.app, but I'm not certain what else to do?? Do I need to manually copy these into the PROJECT folder that I created with Compass.app??  Why are they not manually included?

Thanks!
Jason
